src/js/api/services/UnitOfMeasureApi.js

```diff
@@ -1,6 +1,9 @@
-import { CURRENCIES_OPTIONS } from 'api/urls';
+import { CURRENCIES_OPTIONS, UNIT_OF_MEASURE_OPTIONS } from 'api/urls';
 import apiClient from 'utils/apiClient';
 
 export default {
   getCurrenciesOptions: () => apiClient.get(CURRENCIES_OPTIONS),
+  getUnitOfMeasureOptions: (type) => apiClient.get(UNIT_OF_MEASURE_OPTIONS, {
```

Comment:
shouldn't it follow our current convention with passing `config`, as `params` is not the only property we can pass to axios?